demo-extended/build.gradle

```diff
@@ -30,4 +30,9 @@ dependencies {
     compile project(':library')
     compile 'com.android.support:appcompat-v7:23.1.0'
     compile 'com.android.support:design:23.1.0'
+
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.mockito:mockito-core:1.10.19'
+    testCompile 'org.easytesting:fest-assert-core:2.0M10'
+    testCompile 'org.robolectric:robolectric:2.4'
```

Comment:
We used Robolectric in `UrlFinderTest` and `UrlSpanFactoryTest` otherwise the test would fail since their had no idea about the Android components such as `UrlSpan`s or `Patterns`.